### PR TITLE
SEO audit

### PR DESCRIPTION
**Pull request by natke**
**Description**: SEO audit of the ONNX Runtime docs website.

- Audit titles, descriptions, h1s and first paras. 
- Add redirects for URLs that were moved.
- Generally use ONNX Runtime instead of ORT

Staged here: https://natke.github.io/onnxruntime/docs/

### CODE REVIEW

File: docs/build/reduced.md [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
 ---
 
-# Build ORT with reduced size
+# Reduce the size of the ONNX Runtime build
 {: .no_toc }
 
 For applications where package binary size is important, ONNX Runtime provides options to reduce the build size with some functional trade-offs.
 
-To reduce the compiled binary size of ONNX Runtime, the operator kernels included in the build can be reduced to just the kernels required by your model/s.
+To reduce the compiled binary size of ONNX Runtime, the build can be configuted to only include the operator kernels needed by your model(s).
```
Comment (prasanthpul) [inline docs/build/reduced.md:13]: configuted -> configured

File: docs/execution-providers/ArmNN-ExecutionProvider.md [hunk 2]
```diff
@@ -13,7 +15,7 @@ nav_order: 3
 * TOC placeholder
 {:toc}
 
-[ArmNN](https://github.com/ARM-software/armnn) is an open source inference engine maintained by Arm and Linaro companies. The integration of ArmNN as an execution provider (EP) into ONNX Runtime accelerates performance of ONNX model workloads across Armv8 cores.
+Accelerate performance of ONNX model workloads across Armv8 cores with the ArmMM execution provider. [ArmNN](https://github.com/ARM-software/armnn) is an open source inference engine maintained by Arm and Linaro companies.
```
Comment (prasanthpul) [inline docs/execution-providers/ArmNN-ExecutionProvider.md:18]: ArmMM -> ArmNN

File: docs/execution-providers/MIGraphX-ExecutionProvider.md [hunk 3]
```diff
@@ -1,7 +1,9 @@
 ---
 title: AMD MI GraphX
```
Comment (prasanthpul) [inline docs/execution-providers/MIGraphX-ExecutionProvider.md:2]: MI GraphX -> MIGraphX

File: docs/build/reduced.md (hunk 1 above)
Comment (natke) [reply docs/build/reduced.md]: Fixed, thanks!

File: docs/execution-providers/ArmNN-ExecutionProvider.md (hunk 2 above)
Comment (natke) [reply docs/execution-providers/ArmNN-ExecutionProvider.md]: Fixed, thanks!

File: docs/execution-providers/MIGraphX-ExecutionProvider.md (hunk 3 above)
Comment (natke) [reply docs/execution-providers/MIGraphX-ExecutionProvider.md:2]: Fixed, thanks!

Comment (prasanthpul) [reply docs/execution-providers/MIGraphX-ExecutionProvider.md:2]: title still says MI GraphX

**Review by prasanthpul — APPROVED** [review]

**Pull request merged**